app/ide-desktop/lib/client/src/index.ts

```diff
@@ -19,12 +19,10 @@ import * as paths from 'paths'
 import * as projectManager from 'bin/project-manager'
 import * as security from 'security'
 import * as server from 'bin/server'
-<<<<<<< HEAD
 import opener from 'opener'
 import { DEEP_LINK_PROTOCOL } from '../shared'
-=======
 import * as process from 'process'
->>>>>>> develop
+
 const logger = content.logger
```

Comment:
This PR is probably wrongly rebased. This might be a display error in GitHub caused by force-pushes (I've seen something like that already). Anyway, it's impossible to review the PR until we have correct diff here. I assume that this PR does NOT remove merge conflicts from the code, so something is wrong here. Please check it out, and ping me when it will be correctly applied :)